# IntensityRange2D 激光点集合
int64 timestamp
string frame_id
IntensityPoint2D[] data
